### ISSUE TITLE
Authentication (log in / password reset) fields are not full width

### ISSUE THREAD

**Issue by lb-** [wagtail/wagtail#9440]
### Issue Summary

The log in / auth pages have got an alignment issue with the fields - they are no longer taking up the full container width.

<img width="1740" alt="Screen Shot 2022-10-20 at 10 34 20 pm" src="https://user-images.githubusercontent.com/1396140/196950284-8748f322-fb03-48bb-8378-be9d95337bf6.png">

### Steps to Reproduce

1. Go to the log in page or password reset page
2. Expected: The fields (username/password) should take up the full width of the container and be visually aligned with the log in button
3. Actual: The fields are inset from the left by about `2rem`

### Technical details

-   Python version: 3.9
-   Django version: 4.1
-   Wagtail version: 4.2.dev
-   Browser version: Firefox 15
-  Looks like this change happened for fixing admin fields in d96510949acce6e65073b8eea5ae9553c3eb85a6 - https://github.com/wagtail/wagtail/pull/9164
- We will probably need to add a reset for this padding in `client/scss/layouts/_login.scss` 

Current
```
  .button,
  .w-field__label {
    width: 100%;
    font-size: theme('fontSize.16');
    font-weight: theme('fontWeight.bold');
  }

```

Proposed

```
  .button,
  .w-field__label {
    width: 100%;
    font-size: theme('fontSize.16');
    font-weight: theme('fontWeight.bold');
    padding: 0; // or padding-inline-end: 0;
  }
```

**Comment by xomcodes** [comment]
@lb i would love to work on this

**Comment by lb-** [comment]
Go for it. Please can you try to reproduce the issue first and add a screenshot/ comment - then we can remove the unconfirmed label.

**Comment by xomcodes** [comment]
Okay. I would do that. Thank you.

**Comment by xomcodes** [comment]
@lb I have confirmed the issue. The fields(username/password) are not taking  up the full width of the container just like that of the sign in button.

![issh authentication](https://user-images.githubusercontent.com/82041329/197282517-73ac536e-55a3-4411-a03f-026dc89df2f4.jpg)

**Comment by xomcodes** [comment]
Fixed in  #9465
